src/components/item-view/data-section/ImpactRadar.js

```diff
@@ -104,7 +107,6 @@ const ImpactDonut = ({ item }) => {
           />
         </VictoryChart>
       </div>
-  </div>
   );
 };
 
```

Comment:
Hangs off the bottom of the modal, but I imagine there's more work left to do on that anyway 👍